qa/workunits/rbd/rbd_benchmark_clay_vs_jerasure.sh

```diff
@@ -0,0 +1,214 @@
+  rbd rm ${CLAY_META_POOL}/${CLAY_IMG} || true
+
+  ceph osd pool delete ${JERASURE_DATA_POOL} ${JERASURE_DATA_POOL} --yes-i-really-really-mean-it || true
+  ceph osd pool delete ${JERASURE_META_POOL} ${JERASURE_META_POOL} --yes-i-really-really-mean-it || true
+  ceph osd pool delete ${CLAY_DATA_POOL} ${CLAY_DATA_POOL} --yes-i-really-really-mean-it || true
+  ceph osd pool delete ${CLAY_META_POOL} ${CLAY_META_POOL} --yes-i-really-really-mean-it || true
+
+  ceph osd erasure-code-profile rm ${JERASURE_PROFILE} || true
+  ceph osd erasure-code-profile rm ${CLAY_PROFILE} || true
+
+  rm -rf /tmp/bench_j /tmp/bench_c
+  rm -rf /home/ubuntu/cephtest/fio || true
+  echo "Cleanup complete."
+}
+trap cleanup EXIT
+
+echo "Checking Ceph cluster health..."
+if ! ceph health | grep -qE "HEALTH_OK|HEALTH_WARN"; then
+  echo "Cluster is not healthy (not HEALTH_OK or HEALTH_WARN). Exiting."
+  ceph health detail
+  exit 1
+fi
+echo "Cluster is healthy. Proceeding with benchmark..."
+
+echo "=== STEP 1: Creating EC Profiles ==="
+
+ceph osd erasure-code-profile set $JERASURE_PROFILE \
+  plugin=jerasure k=4 m=2 crush-failure-domain=osd || true
+
+ceph osd erasure-code-profile set $CLAY_PROFILE \
+  plugin=clay k=4 m=2 d=5 technique=cauchy_good \
+  crush-failure-domain=osd || true
+
+echo "=== STEP 2: Creating Pools ==="
+
+# Jerasure
+ceph osd pool create $JERASURE_DATA_POOL 64 64 erasure $JERASURE_PROFILE || true
+ceph osd pool create $JERASURE_META_POOL 64 64 || true
+rbd pool init -p $JERASURE_META_POOL
+ceph osd pool set $JERASURE_DATA_POOL allow_ec_overwrites true
+
+# Clay
+ceph osd pool create $CLAY_DATA_POOL 64 64 erasure $CLAY_PROFILE || true
+ceph osd pool create $CLAY_META_POOL 64 64 || true
+rbd pool init -p $CLAY_META_POOL
+ceph osd pool set $CLAY_DATA_POOL allow_ec_overwrites true
+
+echo "=== STEP 3: Creating RBD Images ==="
+
+rbd create $JERASURE_META_POOL/$JERASURE_IMG --size $IMG_SIZE
+rbd create $CLAY_META_POOL/$CLAY_IMG --size $IMG_SIZE
+
+echo "=== STEP 4: Running rbd bench on Jerasure EC Pool ==="
+START_J=$(date +%s)
+rbd bench --io-type $IO_TYPE --io-size $IO_SIZE --io-threads $IO_THREADS \
+  --io-total $IO_TOTAL --io-pattern $PATTERN $JERASURE_META_POOL/$JERASURE_IMG | tee jerasure_bench.log
+END_J=$(date +%s)
+J_TIME=$((END_J - START_J))
+
+echo "=== STEP 5: Running rbd bench on CLAY EC Pool ==="
+START_C=$(date +%s)
+rbd bench --io-type $IO_TYPE --io-size $IO_SIZE --io-threads $IO_THREADS \
+  --io-total $IO_TOTAL --io-pattern $PATTERN $CLAY_META_POOL/$CLAY_IMG | tee clay_bench.log
+END_C=$(date +%s)
+C_TIME=$((END_C - START_C))
+
+# === Install FIO ===
+echo "Installing fio..."
+if [[ -f /etc/debian_version ]]; then
+  sudo apt-get update
+  sudo apt-get install -y git gcc make librados-dev librbd-dev zlib1g-dev libaio-dev
+  git clone -b master https://github.com/axboe/fio.git /home/ubuntu/cephtest/fio
+  cd /home/ubuntu/cephtest/fio
+  ./configure
+  make
+  sudo make install
```

Comment:
What is the reason for building fio instead of just installing a package like it's done for `/etc/redhat-release` case?